src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.BaseMotorController; 
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX; // internal Falcon motors
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import edu.wpi.first.wpilibj.XboxController; // controller input 
+
+import com.revrobotics.CANSparkMax; // front intake motor
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.CANSparkMax.IdleMode;
+
+import edu.wpi.first.wpilibj.Servo;
+
+import static frc.robot.Constants.IntakeConstants.*;
+
+
+public class Intake extends SubsystemBase {
+    
+    private final CANSparkMax front = new CANSparkMax(FRONT_MOTOR, MotorType.kBrushless); // front motor for intake
+    private final WPI_TalonSRX right = new WPI_TalonSRX​(RIGHT_MOTOR); // right roller between intake and carriage
+    private final WPI_TalonSRX left = new WPI_TalonSRX​(LEFT_MOTOR); // left roller between intake and carriage
+
+    Servo intake_servo = new Servo(INTAKE_SERVO);
+
+    public boolean intake_down = false;
+    public boolean intake_on = false;
```

Comment:
i think we should put states at the very top for readability

also, i think drivers would appreciate being able to reverse the intake or go slower, which is why we usually put the intake on (two) triggers

Reply:
based on our intake tests, full power is a little too fast, so we'll definitely want to be able to set a variable power as the state. 

We'll proably even want to scale that power before setting the state in RobotContainer so full trigger sets like 0.5 power